src/frameworks/i18next/parser/i18n-next.ts

```diff
@@ -51,7 +51,7 @@ export function i18NextParser({ mockSourceFile, sourceFile }: ParserInput<{}>):
                         content: variable,
                         position: addPosition(value_position, [0, pos + prefix.length + StartQuoteLength]),
                     },
-                    propertyAccess.length > 1 ? 'object' : 'string',
+                    propertyAccess.length > 1 ? 'object' : 'string | number',
```

Comment:
The problem here is you don't know if it really means a number in this position just by the i18n string, therefore I'd prefer to require force type convention here.